.github/workflows/ci.yml

```diff
@@ -481,6 +494,9 @@ jobs:
             packages/qwik-labs/vite/
             packages/qwik-labs/package.json
           if-no-files-found: error
+      - run: pnpx pkg-pr-new publish --compact ./packages/qwik/dist ./packages/qwik-city/dist dist-dev-eslint-plugin-qwik dist-dev-create-qwik
```

Comment:
What makes these dist-dev directories?

Reply:
the upload-artifacts task from the job before